CMake/elastixVersion.cmake

```diff
@@ -2,3 +2,5 @@
 set(ELASTIX_VERSION_MAJOR "5")
 set(ELASTIX_VERSION_MINOR "0")
 set(ELASTIX_VERSION_PATCH "1")
+
+set(ELASTIX_VERSION "${ELASTIX_VERSION_MAJOR}.${ELASTIX_VERSION_MINOR}.${ELASTIX_VERSION_PATCH}")
```

Comment:
Looks like we might also use this `ELASTIX_VERSION` CMake variable for the C++ `#define ELASTIX_VERSION_STRING`, now in: https://github.com/SuperElastix/elastix/blob/5.0.1/Core/elxVersionMacros.h.in#L33  😃